.storybook/DocsDecorator/index.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { type JSX } from 'react';
```

Comment:
Do we still need to import `React` here for storybook if we're removing it from imports in components below?

i.e in the Next app at `ws-nextjs-app/pages/[service]/send/[id]/MessageBox/ErrorSummaryBox.tsx`

Reply:
Yea we'll need to here. Other components that don't import will likely be importing something like:
```
/** @jsx jsx */
import { jsx } from '@emotion/react';
```
which means Emotion will be taking control of JSX rendering.